### PR TITLE
feat: add corrupted bar

### PR DESCRIPTION
**Pull request by byBackfish**
Adds the corrupted bar (warrior ability) 
image

### CODE REVIEW

File: src/main/java/com/wynntils/modules/utilities/overlays/hud/CorruptedBarOverlay.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                break;
+            case Rune:
+                drawDefaultBar(-1, 8, 148, 163, textColor);
+                break;
+        }
+    }
+
+    private void drawDefaultBar(int y1, int y2, int ty1, int ty2, CustomColor cc) {
+        if (OverlayConfig.CorruptedBar.INSTANCE.overlayRotation == OverlayRotation.NORMAL) {
+            drawString(get(CharacterData.class).getCorruptedProgressPercent() + " ۞ " + maxCorruptedPercent, textPositionOffset.a  - (81 - OverlayConfig.CorruptedBar.INSTANCE.width), textPositionOffset.b, cc, SmartFontRenderer.TextAlignment.MIDDLE, OverlayConfig.CorruptedBar.INSTANCE.textShadow);
```
Comment (Incompleteusern) [inline src/main/java/com/wynntils/modules/utilities/overlays/hud/CorruptedBarOverlay.java]: Find a new symbol, this is the same as awakened. Maybe something like "☫" or "☦" works?

**Review by Incompleteusern — COMMENTED** [review]
One thing

File: src/main/java/com/wynntils/modules/utilities/overlays/hud/CorruptedBarOverlay.java (hunk 1 above)
Comment (byBackfish) [reply src/main/java/com/wynntils/modules/utilities/overlays/hud/CorruptedBarOverlay.java]: was changed already in https://github.com/Wynntils/Wynntils/pull/595/commits/837448d38e1aa69fa99a26b3cb0f372dcabb6a30

Comment (byBackfish) [reply src/main/java/com/wynntils/modules/utilities/overlays/hud/CorruptedBarOverlay.java]: to a skull symbol, i though a skull might be the best thing to have as the symbol

Comment (Incompleteusern) [reply src/main/java/com/wynntils/modules/utilities/overlays/hud/CorruptedBarOverlay.java]: Yeah, true

**Review by Incompleteusern — APPROVED** [review]

**Review by magicus — APPROVED** [review]

**Pull request merged**